### DISCUSSION TITLE
请问大佬可以反代实现steam论坛访问吗？

### DISCUSSION THREAD

**Discussion by leafsnow** [SeaHOH/GotoX#206] [一般]
https://steamcommunity.com/

**Comment by SeaHOH** [comment]
为什么要反代呢，客户端不能设置代理吗？
这个工具可以设置成反代 (监听 443)，但没有专门优化反代体验。Adnroid App 肯定不行，Windows 客户端估计可以使用导入系统的证书。

**Comment by 2378** [comment]
这不是反不反代的问题。
steam的策略就这样，单一IP账号一多直接封，什么加速器、VPN被封的多了
原因无外乎，打击黑产灰产，保证游戏公平，维护区域销售，等等
用自己的VPS上没有任何问题

**Comment by SeaHOH** [comment]
主要看个人需求。
对延迟要求高当然不能用我这个工具搞，如楼上所言建议自己搞一个适合的 VPS 开 VPN，个位数的账号@单 IP 估计问题不大。
如果不在乎加速是可以本地直连的，UDP 不论延迟但基本是畅通的，HTTP 也无需通过远程代理。
问题是，为了 HTTP 直连个别域名 (比如 community 主域名) 中间人要修改 SNI，并使用自签证书中继连接。

**Comment by 2378** [comment]
@SeaHOH 海大，隔壁可炸锅了，GAE没绑卡的ID全军覆没，这次狼是真来了 :worried:

**Comment by SeaHOH** [comment]
搞不懂非要用 GAE 的道理，那么多公共 ID，怎么可能全部都绑上，要用只好绑自己的，何苦呢。

**Comment by 2378** [comment]
反正两年前就停用GAE了，但看着有人屁颠屁颠的 把自己的卡号电话奉上，心中一阵感慨，不知是福是祸啊 :cold_sweat:
